tests/unit/test_wloader.py

```diff
@@ -275,6 +276,13 @@ def testLoadWholeConfig(self):
 c "d d"
 </a>
 # a
+
+
 """
-        node = ListNode.parse(text, self.parser)
+        t = tempfile.mkdtemp()
```

Comment:
I wonder if `with mkdtemp():` would clean up automatically?

Reply:
`mkdtemp` isn't a context manager like the file equivalent, unfortunately! I also thought about using the `NamedTemporaryFile` context manager, but I'd like to be able to close the filehandle without deleting the file, so `loads` can re-open it.